frontend/src/main/webapp/triplist.js

```diff
@@ -0,0 +1,20 @@
+function tripData() {
+  var trips = [
+    {'name': 'Uk trip'}, {'name': 'Bora Bora trip'}, {'name': 'Mexico trip'},
+    {'name': 'Japan trip'}, {'name': 'European trip'}
+  ];
+  getTripList(trips);
+}
+
+function getTripList(trips) {
```

Comment:
I think this function could have a better name since it's not returning a trip list like the get implies - maybe something like setTripListPage